CNAME

```diff
@@ -1 +0,0 @@
-sfcrimedata.org
```

Comment:
for sure we want to keep this guy

Reply:
Hugo, Yes, This is the only thing keeping me from hitting the "PULL" button right now.

However, I'm not clear that the presence of the CNAME file in the root is still required for Github Pages hosting anymore. Github Help says to set the custom domain in Settings, and does not specify that CNAME is required (see Ref: below). There is still Github Help for troubleshooting if your CNAME file is wrongly configured, but no "first setup" instruction for CNAME. I can't find a clear statement from Github about whether CNAME is required, or is deprecated in favor of using Settings.

That said, the CNAME file is a PITA. It's only valid on the "live" site. It causes an error to be thrown for developers testing the site on their personal Github Pages, (like David's hosting at https://drothschild.github.io/sf-crime-data).

Thoughts? Does anyone have clearer info from Github?

If not, I may "test in production" some evening. =:o If the pull "as is" works without CNAME, great. If it causes the site to die, CNAME can easily be re-added.

Ref: https://help.github.com/articles/adding-or-removing-a-custom-domain-for-your-github-pages-site/